src/openforms/dmn/contrib/camunda/plugin.py

```diff
@@ -122,3 +124,12 @@ def get_definition_xml(definition_id: str, version: str = "") -> str:
 
     def check_config(self):
         check_config(self)
+
+    def get_decision_definition_parameters(
+        self, definition_id: str, version: str = ""
+    ) -> DMNIntrospectionResult:
+        xml_doc = self.get_definition_xml(definition_id, version)
+
+        parser = Parser(xml=xml_doc.encode("utf-8"))
+
+        return parser.extract_parameters(definition_id)
```

Comment:
You should now be able to replace this with:

```py
from django_camunda.dmn import get_dmn_parser

with get_client() as client:
    camunda_id = _get_decision_definition_id(client, definition_id, version)
    parser = get_dmn_parser(dmn_id=camunda_id, client=client)
return parser.extract_parameters(definition_id)
```
right?

The retrieval of the DMN XML is already done in `get_dmn_parser`, no need for this code to still exist here.